config/clusters/nasa-veda/common.values.yaml

```diff
@@ -96,6 +96,17 @@ basehub:
                   slug: pangeo
                   kubespawner_override:
                     image: public.ecr.aws/nasa-veda/nasa-veda-singleuser:5068290376e8c3151d97a36ae6485bb7ff79650b94aecc93ffb2ea1b42d76460
+                    init_containers:
+                      - name: nasa-veda-singleuser-init
```

Comment:
Can you add a comment here describing what is being done?

And to update the other two profiles as well if this is to be updated?

Reply:
Also a link to the github source of this image would help!

Reply:
updated. 

I shouldn't have to update the two other profiles b/c the one I updated uses the anchor ref `image: &image_options` and the other profiles reuse that via `image: *image_options` correct?

Reply:
hmm, where is this qgis image?

Reply:
I feel like I'm missing something 😄

Reply:
added it to `unlisted_choice` as well so I think we're covered

Reply:
@ranchodeluxe I mixed this up with GHG which has another profile option for qgis, oops! Sorry.